### PR TITLE
Editorial: xref cleanup

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -59,11 +32,12 @@
...
       }
     },
-    wgPatentURI:  "https://www.w3.org/2004/01/pp-impl/45211/status"
+    implementationReportURI: "http://wpt.fyi/requestidlecallback",
+    xref: "web-platform",
   };
   </script>
 </head>
-<body>
+<body data-cite="page-visibility-2">
```
Comment (yoavweiss) [inline index.html]: What does that data-cite do? Why page visibility?

**Review by yoavweiss — COMMENTED** [review]
Looks good. One question tho

File: index.html (hunk 1 above)
Comment (marcoscaceres) [reply index.html]: Down below, there is a note that talks about:
> For example, if the {{Document}} is [=Document/hidden=]...  

I'll move the data-cite to the note, so it's more obvious.

**Review by yoavweiss — APPROVED** [review]
LGTM

**Comment by marcoscaceres** [comment]
Thanks again for the speedy review!

**Pull request merged**